src/components/banner.js

```diff
@@ -1,22 +1,23 @@
-import axios from 'axios';
 import React, {useState, useEffect, useCallback} from 'react';
 // import {useTranslation} from 'react-i18next';
 
+const DATA_URL = 'https://api.covid19india.org/website_data.json';
```

Comment:
Just wanted to clarify in case you mistook this for the landing page's data, those endpoints would be:
- https://api.covid19india.org/v3/min/data.min.json
- https://api.covid19india.org/v3/min/timeseries.min.json